.github/workflows/deploy.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     - name: install
```

Comment:
@everettraven we have also the changes in : https://github.com/operator-framework/operator-sdk/pull/5835/files

Should not be done here as well?
JUst the bumps in the SDK makefile

Reply:
It does not requires changelog

Reply:
I think we had them separate for downstream JIRA ticket purposes. I think the goal was to do as much as possible in parallel. Since the PR is already up I am going to just have that one merge as a follow up.